aiida/cmdline/params/options/interactive.py

```diff
@@ -103,14 +105,15 @@ def _get_default(self, ctx):
     def prompt_func(self, ctx):
         """prompt function with args set"""
         return click.prompt(
-            self._prompt,
+            click.style(self._prompt, fg=self.PROMPT_COLOR),
+            prompt_suffix = click.style(': ', fg=self.PROMPT_COLOR),
```

Comment:
spaces between keyword and value, was this not caught by the linter? Is it in the pre-commit config?